_posts/events/2017-05-03---azure-machine-learning-and-securing-azure-assets.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: single
+title: "2017-05-03 - Azure Machine Learning and securing Azure assets"
+date: 2017-05-03 13:37:00 +0000
+comments: true
+published: true
+categories: ["events"]
+tags: ["Events"]
+alias: ["/events/2017-05-03---azure-machine-learning-and-securing-azure-assets"]
+author: Tom Kerkhove
+redirect_from:
+ - /events/2017-05-03---azure-machine-learning-and-securing-azure-assets.html
+---
+
+Hello Customer is happy to host an event in the Ghelacomo arena to talk about finding nuggets with Azure Machine Learning (ML) Studio and Mike Marting will talk about how and where azure security works and what you can do to secure your Azure resources.
+
+## Sessions
+### Finding nuggets with Azure ML Studio
+Currently there are not a lot of point-and-click data science tools, Azure ML Studio is among the first with a very specific offering. During this session we will make a text analysis exercise on azure ml studio and see where the pitfalls are. After the exercise we will discuss the advantages and disadvantages of using Azure ML Studio.
+
+<img src="/assets/media/speakers/tijl-carpels.png" alt="Tijl Carpels" align="left" width="100" height="100" style="margin-right: 20px;">**Speaker:** *As a Data Scientist, Tijl Carpels is designing and implementing Natural Language Processing pipelines at Hello Customer. His interests range from distributed computing to machine learning and developing chatbots.*
+
+### Enter The Matrix: Securing Azure's Assets
```

Comment:
This one looks funky but I'm not a web expert. Feel free to let me know how I can fix it